### ISSUE TITLE
PMID:37615341

### ISSUE THREAD

**Issue by PCarme** [pombase/fypo#4522]
New term : Normal replication fork arrest at RTS1 barrier
A cellular process phenotype observed in the vegetative growth phase of the life cycle in which replication fork arrest at the RTS1 barrier (normally located in the mating-type region) is normal (i.e. indistinguishable from wild type).

**Comment by ValWood** [comment]
I think the "RTS1 barrier" might be in  rDNA ant m-t locus? do you know?

if so, use
normal replication fork arrest at mating-type locus
and I will add synonyms/ improve definition

(I added a parent to group terms)

**Comment by PCarme** [comment]
> I think the "RTS1 barrier" might be in rDNA and the m-t locus? do you know?
> 
> if so, use normal replication fork arrest at mating-type locus and I will add synonyms/ improve definition
> 
> (I added a parent to group terms)

Not really. When describing their system, they cite one of their previous paper, in which I found "To introduce the RTS1 sequence into 4 loci analysed in Supplementary Fig. 1, the kan cassette from pFAX-kanMX6 was amplified by the
following primers: 461/462 (mns1 locus), 463/464 (tel2 locus), 465/466 (mek1 locus), and F3/R4 (spo9 locus).[...] To create the optimised ChrII locus (Fig. 1) the 10xrRFB sequence was integrated downstream of RTS1 barrier. Primers F29 and R30 were used to amplify ura4+ and the product was transformed into KA2 and checked for insertion onto the ChrII between SPBC36.10.1 and SPBC36.11.1 to form strain KA39."
 The position described for the 10xrRFB is close to the spo9 locus, so I assume they made this construct on the RTS1 inserted at the spo9 locus, which is located almost 1.3kb upstream of the mating-type locus.

**Comment by ValWood** [comment]
Hmm, I see. We might need to model this differently.

I don't think we can call it "Normal replication fork arrest at RTS1 barrier" because it isn't  at its normal location. In a WT situation there would be no arrest at this point.

I will take a look... what is the PMID of the previous paper?

**Comment by PCarme** [comment]
PMID: 33568651

**Comment by ValWood** [comment]
I'm a bit confused here.
Sometimes we have
normal site-specific DNA replication termination at RTS1 barrier (FYPO:0005354)

and in other places we refer to "arrest"
https://www.pombase.org/term/FYPO:0003088
i.e decreased replication fork arrest at RTS1 barrier (FYPO:0006603)
and in other places to  "site-specific termination"
https://www.pombase.org/term/FYPO:0005355

 should be standardized and synonyms added (Q use arrest or termination?)

I could edit
normal replication fork arrest at mating-type locus
A cellular process phenotype observed in the vegetative growth phase of the life cycle in which replication fork arrest at the mating-type locus is normal (i.e. indistinguishable from wild type).

to 
normal replication fork arrest at RTS1 barrier
A cellular process phenotype observed in the vegetative growth phase of the life cycle in which replication fork arrest at the RTS1 barrier  is normal (i.e. indistinguishable from wild type). Note that the RTS1 barrier usually occurs in the mating type locus, but might be assayed at ectopic loci to maximise the distance that a restarted fork travels.

and in this case you can add some "background" to say RTS1@spo9 locus?

**Comment by ValWood** [comment]
TC:
I would say arrest, since replication can be restarted (all be it by a different set of proteins rather than the usual replisome). A terminated event is over for good (although the tautology is complex). In any case one or the other but not both.

so I will change these to arrest (will check sources)
https://www.pombase.org/term/FYPO:0005355

**Comment by ValWood** [comment]
For now, I have revised the existing term as suggested in
https://github.com/pombase/fypo/issues/4522#issuecomment-2044744360

I did not implement TC suggestion at the moment because the papers which use the termination term do appear to refer to termination.


GO:0071171   [ JSON](https://www.ebi.ac.uk/QuickGO/services/ontology/go/terms/GO:0071171/complete)
site-specific DNA replication termination at RTS1 barrier
is a child of replication termination, so logical definitions should place all terms under  "DNA replication termination" terms anyway.
Can consider standardizing ion the future but might need to check the papers using the replication terms first. I'm worried that  based on Tony C's comment about 'restart' the GO parentage may not be appropriate, so this can wait until we are looking more closely at this area.
